lib/apm.js

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var path = require('path');
+
+// Default paths for APM installation
+var apmLinuxPath = path.join('/opt', 'ibm', 'apm', 'agent');
+var apmWindowsPath = path.join('C:', 'IBM', 'APM');
+
+module.exports = function enablePerformanceMetrics() {
+  var defaultApmPath = (process.platform === 'win32') ? apmWindowsPath :
+    apmLinuxPath;
+  var apmPath = process.env.STRONGLOOP_APM_PATH || defaultApmPath;
+  var pluginPath = path.join('lx8266', 'nj', 'bin', 'plugin');
+  var pluginBasePath = path.join(apmPath, pluginPath);
```

Comment:
this could just be

```
var knj = path.resolve(apmPath, 'lx8266', 'nj', 'bin', 'plugin', 'knj_methodtrace');
require(knj);
```
I think its even less < 80 columns.